cli/bench/main.rs

```diff
@@ -401,38 +403,40 @@ fn cargo_deps() -> usize {
 
 #[derive(Serialize)]
 struct BenchResult {
-  created_at: String,
-  sha1: String,
-  binary_size: HashMap<String, u64>,
-  bundle_size: HashMap<String, u64>,
-  cargo_deps: usize,
   // TODO(ry) The "benchmark" benchmark should actually be called "exec_time".
   // When this is changed, the historical data in gh-pages branch needs to be
   // changed too.
   benchmark: HashMap<String, HashMap<String, f64>>,
-  throughput: HashMap<String, f64>,
+  binary_size: HashMap<String, u64>,
+  bundle_size: HashMap<String, u64>,
+  cargo_deps: usize,
+  created_at: String,
+  max_latency: HashMap<String, f64>,
   max_memory: HashMap<String, u64>,
+  lsp_exec_time: HashMap<String, u128>,
   req_per_sec: HashMap<String, u64>,
-  max_latency: HashMap<String, f64>,
-  thread_count: HashMap<String, u64>,
+  sha1: String,
   syscall_count: HashMap<String, u64>,
+  thread_count: HashMap<String, u64>,
+  throughput: HashMap<String, f64>,
```

Comment:
sha1 and created_at are special metadata, and should probably remain at the top for easy viewing